potodo/po_file.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, path: Path):
         ] = self.pofile.untranslated_entries()
         self.untranslated_nb: int = len(self.untranslated_entries)
 
+        self.entries_count: int = self.translated_nb + self.untranslated_nb
```

Comment:
```suggestion
        self.entries_count: int = len([e for e in self.pofile if not e.obsolete])
```

from polib itself, which computes percent_translated as:

```python
def percent_translated(self):
    """
    Convenience method that returns the percentage of translated
    messages.
    """
    total = len([e for e in self if not e.obsolete])
    if total == 0:
        return 100
    translated = len(self.translated_entries())
    return int(translated * 100 / float(total))
```

so to ensure the result is consistent, use the same values (you use the .percent_translated on line 47, and you'll use entries_count as a weight for percent_translated later)